src/bluehawk/parser/validator.test.ts

```diff
@@ -21,6 +21,7 @@ describe("validator", () => {
     "code-block": makeBlockTag<IdRequiredAttributes>({
```

Comment:
Working on an old branch?

Reply:
those changes still in review - https://github.com/mongodb-university/Bluehawk/pull/98